doc/user/user/sistr/index.md

```diff
@@ -26,51 +26,37 @@ The SISTR pipeline can bet set up to run using two separate methods.
 
 ### 1. Automated Execution
 
-The SISTR pipeline can be set to run automatically on upload of new sequencing data to particular projects. This can be set either on the creation of a new project, or from the project settings page.
+The SISTR pipeline can be set to run automatically on upload of new sequencing data to particular projects.
 
-a. **Creation of New Project**
-
-   On creation of a new project, the below option can be selected to enable the automated execution of SISTR on upload of new data to this project.
-
-   ![new-project-sistr.png][]
-
-b. **Project Settings**
+**Project Settings**
 
    Automated SISTR analysis can also be enabled (or disabled) after a project is created from the project **Settings** page.
 
-   ![project-settings-sistr.png][]
+   ![project-settings-sistr.png](images/project-settings-sistr.png)
 
-If automated execution of SISTR has been enabled for a project, then a SISTR analysis will be scheduled for execution on the upload of new sequencing data.  The results are accessible from the particular sample page.
+If automated execution of SISTR has been enabled for a project, then a SISTR analysis will be scheduled for execution on the upload of new sequencing data.  The results are accessible from the particular `Project` > `Analyses` page.
 
-![sistr-typing-sample.png][]
+![sistr-typing-analysis-page.png](images/sistr-typing-analysis-page.png)
 
 Clicking the **Automated SISTR Typing** link brings you to the appropriate analysis page for SISTR.
 
-![sistr-typing-status.png][]
+![sistr-typing-status.png](images/sistr-typing-status.png)
 
 ### 2. Manual Execution
 
 To execute SISTR manually, please refer to the [IRIDA/SISTR Tutorial][].
 
 ## SISTR Results
 
-### Status of `PASS`
-
-A successfull SISTR run (with status of `PASS`) should produce the following page as output.
-
-![sistr-results.png][]
-
-### Status of `WARNING`
-
-A SISTR run with a status of `WARNING` should produce the below output.
+A successful SISTR run should produce the following page as output. There are three possible Quality Control Statuses (Pass, Warning, and Fail)
 
-![sistr-results-warning.png][]
+![sistr-results.png](images/sistr-results.png)
```

Comment:
Image is too big to see on the docs page.  Crop it or reshoot with a scaled browser please.

Reply:
Updated in 6d265b032635f8e0f76f98eddb88a30e899bf36f